test_cases/address_matching.json

```diff
@@ -5,7 +5,7 @@
   "tests": [
     {
       "id": 1,
-      "status": "pass",
+      "status": "fail",
```

Comment:
add issue property to test and reference https://github.com/pelias/pelias/issues/424